### PR TITLE
Update Editors to reflect contributions.

### PR DESCRIPTION
**Pull request by msporny**
This PR addresses issue #1273 by updating the list of Editors to [reflect contributions](https://github.com/w3c/vc-data-model/graphs/contributors?from=2022-03-28&to=2023-11-18&type=c) made to date. It also removes @awoie as a CODEOWNER since he has stepped down as an Editor of the specification.

@TallTed, please provide your w3cid in the PR (or @iherman, let me know what it is).

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/pull/1354.html" title="Last updated on Nov 20, 2023, 2:34 PM UTC (ca2e2b1)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/1354/b3cff22...ca2e2b1.html" title="Last updated on Nov 20, 2023, 2:34 PM UTC (ca2e2b1)">Diff</a>

### CODE REVIEW

**Comment by iherman** [comment]
The W3C ID of @TallTed is 42501

**Review by iherman — COMMENTED** [review]
I guess the proper thing for me is to abstain from a review as far as I am concerned, but I am in favor of adding Ted.

File: index.html [hunk 1]
```diff
@@ -72,6 +72,12 @@
           { name: "Manu Sporny", url: "https://www.linkedin.com/in/manusporny/",
             company: "Digital Bazaar", companyURL: "https://digitalbazaar.com/",
             note: "v1.0, v1.1, v2.0", w3cid: 41758},
+          { name: "Ted Thibodeau", url: "http://id.myopenlink.net/dataspace/person/tthibodeau",
+            company: "OpenLink Software", companyURL: "https://www.openlinksw.com/",
+            note: "v2.0", w3cid: 1},
```
Comment (iherman) [inline index.html:77]:
```suggestion
            note: "v2.0", w3cid: 42501},
```

**Review by dlongley — APPROVED** [review]

File: index.html [hunk 2]
```diff
@@ -72,6 +72,12 @@
           { name: "Manu Sporny", url: "https://www.linkedin.com/in/manusporny/",
             company: "Digital Bazaar", companyURL: "https://digitalbazaar.com/",
             note: "v1.0, v1.1, v2.0", w3cid: 41758},
+          { name: "Ted Thibodeau", url: "http://id.myopenlink.net/dataspace/person/tthibodeau",
```
Comment (TallTed) [inline index.html:75]:
```suggestion
          { name: "Ted Thibodeau Jr", url: "https://github.com/TallTed",
```

**Review by decentralgabe — APPROVED** [review]

**Review by OR13 — APPROVED** [review]

**Comment by msporny** [comment]
Substantive, multiple reviews, changes requested and made, no objections, merging.

**Pull request merged**